### PR TITLE
Add EtherType Enum

### LINKED ISSUE #460: Add EtherType ENUM
Add an ENUM with IEEE Ethertypes in order to make the code more readable and more constant.

### PR DESCRIPTION
**Pull request by diraol**
Fix #460

### CODE REVIEW

File: pyof/foundation/network_types.py [hunk 1]
```diff
@@ -3,14 +3,48 @@
...
 from pyof.foundation.exceptions import PackException, UnpackException
 
-__all__ = ('ARP', 'Ethernet', 'GenericTLV', 'IPv4', 'VLAN', 'TLVWithSubType',
-           'LLDP')
+__all__ = ('ARP', 'Ethernet', 'EtherType', 'GenericTLV', 'IPv4', 'VLAN',
+           'TLVWithSubType', 'LLDP')
+
+
+# NETWORK CONSTANTS AND ENUMS
+class EtherType(IntEnum):
```
Comment (renanrodrigo) [inline pyof/foundation/network_types.py]: I think we can add the MPLS unicast/multicast values (0x8847/0x8848),

File: pyof/foundation/network_types.py [hunk 2]
```diff
@@ -3,14 +3,48 @@
...
+    #: with NNI compatibility[8]
+    VLAN = 0x8100
+    #: Internet Protocol Version 6 (IPv6)
+    IPV6 = 0x86DD
+    #: Ethernet flow control
+    ETHERNET_FLOW_CONTROL = 0x8808
+    #: Link Layer Discovery Protocol (LLDP)
+    LLDP = 0x88CC
+    #: VLAN-tagged (IEEE 802.1Q) frame with double tagging
+    VLAN_DOUBLE_TAG = 0x9100
```
Comment (renanrodrigo) [inline pyof/foundation/network_types.py]: Maybe the name VLAN_QINQ would make sense here?
Besides that, 0x9100 is an outdated value for an old protocol - the value for QinQ S-TAGs is 0x88a8

File: pyof/foundation/network_types.py (hunk 1 above)
Comment (diraol) [reply pyof/foundation/network_types.py]: Done! =)

File: pyof/foundation/network_types.py (hunk 2 above)
Comment (diraol) [reply pyof/foundation/network_types.py]: Thanks for the suggestion, VLAN_QINQ added (and value fixed)

**Pull request merged**